src/pages/Signup/Signup.js

```diff
@@ -1,8 +1,136 @@
-import React, { Component } from 'react';
+import React from 'react';
+// import { validationFor } from './Validation';
+import { POST_SIGNUP_API } from '../../../src/config.js';
+import './Signup.scss';
+
+class Signup extends React.Component {
+  state = {
+    name: '',
+    email: '',
+    pw: '',
+    mobile: '',
+  };
+
+  // 입력 함수
+  handleInput = e => {
+    const { name, value } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  // Back이랑 연결하는 fetch 함수
+  requestSignup = () => {
+    fetch(`${POST_SIGNUP_API}`, {
+      method: 'POST',
+      body: JSON.stringify({
+        name: this.state.name,
+        mobile: this.state.mobile,
+        email: this.state.email,
+        password: this.state.pw,
+      }),
+    })
```

Comment:
```suggestion
    const {name,email,pw,mobile} = this.state;
    fetch(`${POST_SIGNUP_API}`, {
      method: 'POST',
      body: JSON.stringify({
        name, mobile, email,
        password:pw,
      }),
    })
```
객체 단축 속성명 이용해서 위와 같이 표현할 수도 있겠네요, password도 state이름을 변경해서 똑같이 맞춰주면 좋겠네요!

Reply:
넵!